contribute/design/oep-process.md

```diff
@@ -0,0 +1,322 @@
+---
```

Comment:
we should probably push these docs under contribute/process directory.

Reply:
Done